source/index.html

```diff
@@ -1,12 +1,118 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Седона</title>
-  </head>
-  <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS, уровень 2</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Седона</title>
+</head>
+
+<body class="page__body">
+  <header class="page-header">
+    <a class="header__logo">
+      <img src="https://via.placeholder.com/102x84" width="102" height="84" alt="Городок Седона">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+      <div class="main-nav__wrapper">
+        <ul class="main-nav__list">
+          <li class="main-nav__item">
+            <a>Главная</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="/source/catalog.html">Фото и видео</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="/source/form.html">Форма отзыва</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="https://htmlacademy.ru/" target="_blank">Html academy</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <main class="page-main">
+    <h1 class="page-main__header">СЕДОНА — небольшой городок в Аризоне, заслуживающий большего!</h1>
+    <section class="features">
+      <div class="features__wrapper">
+        <h2 class="visually-hidden">Преимущества Седоны</h2>
+        <p class="features__intro">Рассмотрим 5 причин, по которым Седона круче, чем Гранд Каньон!</p>
+        <ul class="features__list">
+          <li class="features__item">
+            <div class="features__card features__card--main">
+              <h3 class="features__title">Настоящий городок</h3>
+              <p features__number>— №1 —</p>
+              <p features__description>Седона не аттракцион для туристов, там течет своя жизнь</p>
+            </div>
+            <div class="features__image features__image--city"></div>
+            <ul class="features__recommendation-list">
+              <li class="recommendation-list__item recommendation-list__item--housing">
```

Comment:
блок можно вкладывать в блок, это норм. Иначе тут `li` получается сирота, т.к. блока `recommendation-list` не существует
```suggestion
            <ul class="recommendation-list">
              <li class="recommendation-list__item recommendation-list__item--housing">
```

Чтобы определить другие ошибки БЭМ-нотации документа можешь воспользоваться [генератором HTML-дерева](http://yoksel.github.io/html-tree/), он сгенерит дерево классов и укажет на проблемы, если они есть.